tests/MongoDB.Driver.Examples/InsertDataWithEncryptedFieldsWithAutoEncryption.cs

```diff
@@ -0,0 +1,293 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using MongoDB.Bson;
+using MongoDB.Driver.Encryption;
+using Xunit;
+
+namespace MongoDB.Driver.Examples
+{
+    public class InsertDataWithEncryptedFieldsWithAutoEncryption
+    {
+        private static readonly string __connectionString = "mongodb://localhost:27017";
+        private static readonly string __sampleNameValue = "John Doe";
+        private static readonly int __sampleSsnValue = 145014000;
+
+        private static BsonDocument __sampleDocFields =
```

Comment:
private static readonly BsonDocument __sampleDocFields =

You left off `readonly` by mistake